lib/create-app-definition.js

```diff
@@ -23,75 +23,79 @@ ${chalk.red(
 ${err.message}
     `);
 
-    process.exit(1);
+    throw err;
```

Comment:
This now throws and the exist happens once at the end of the main function block. This was namely to make it testable